settings.gradle

```diff
@@ -2,3 +2,6 @@ rootProject.name = 'gtfs-validator'
 include 'core'
 include 'processor'
 include 'main'
```

Comment:
Do we still need `include 'main'` if `include 'main:cli'` and `include 'main:springboot'` are added?

Reply:
Good catch, removed in https://github.com/MobilityData/gtfs-validator/pull/962/commits/755b7eb8a955df68c98c0bfe2703a67f51f301ac.